### PR TITLE
Remove the redundant ISO-8601 text in the readme.

### PR DESCRIPTION
**Pull request by abhishekrb19**
Fixes the redundant "ISO-8601" text in the README.

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

**Review by asdf2014 — APPROVED** [review]
Nice catch!

File: docs/operations/rule-configuration.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Interval load rules have type `loadByInterval`. The following example places one
 
 Set the following properties:
 
-- `interval`: the load interval specified as an ISO-8601 [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
+- `interval`: the load interval specified as an [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```
Comment (vtlim) [inline docs/operations/rule-configuration.md:181]:
```suggestion
- `interval`: the load interval specified as an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```

File: docs/operations/rule-configuration.md [hunk 2]
```diff
@@ -257,7 +257,7 @@ Interval drop rules have type `dropByInterval` and the following JSON structure:
 
 Set the following property:
 
-- `interval`: the drop interval specified as an ISO-8601 [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
+- `interval`: the drop interval specified as an [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```
Comment (vtlim) [inline docs/operations/rule-configuration.md:260]:
```suggestion
- `interval`: the drop interval specified as an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```

File: docs/operations/rule-configuration.md [hunk 3]
```diff
@@ -313,7 +313,7 @@ Interval broadcast rules have type `broadcastByInterval` and the following JSON
 
 Set the following property:
 
-- `interval`: the broadcast interval specified as an ISO-8601 [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
+- `interval`: the broadcast interval specified as an [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```
Comment (vtlim) [inline docs/operations/rule-configuration.md:316]:
```suggestion
- `interval`: the broadcast interval specified as an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) range encoded as a string.
```

**Review by vtlim — APPROVED** [review]
Nit on spacing, otherwise looks good

**Pull request merged**